mpf/devices/combo_switch.py

```diff
@@ -247,7 +247,7 @@ def _switch_state(self, state):
         self.debug_log("New State: %s", state)
 
         for event in self.config['events_when_{}'.format(state)]:
-            self.machine.events.post(event)
+            self.machine.events.post(event, triggering_switch=triggering_switch)
```

Comment:
We could also include the switch name here.

Reply:
Updated! Now includes `triggering_group` with the group number and `triggering_switch` with the switch name.